CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The sections should follow the order `Added`, `Changed`, `Fixed`, and `Removed`.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+- On macOS, use `objc2` when determining whether font smoothing is enabled.
```

Comment:
Should this actually change anything for consumers of this crate?

Reply:
Nope, I'm fine with not having a changelog entry

Reply:
Yeah seems very internal, I don't think a changelog entry is necessary.